nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -137,6 +139,63 @@ convention = "google"
     "S"
 ]
 
+[tool.pymarkdown]
+# Seems to be not support for whitelisting rules: https://github.com/jackdewinter/pymarkdown/issues/1396
+plugins.md001.enabled = false
```

Comment:
Can we use the aliases for these rules instead of the rule numbers? Instead of `plugins.md001.enabled = false` use `plugins.heading-increment.enabled = false`

Do we need to disable all of these rules? `MD002` for example is disabled by default so we probably don't need to disable it here?

Reply:
I'm concerned that this doesn't align with Nautobot core's configuration and we're trying to be consistent across the ecosystem.

Reply:
As mentioned

> This PR is not about styling, it is about functionality. I am catching issues with these two issues on almost weekly basis. This will create a bit more work to get this cookie baked in, however it is fixing actual issues, not just style points.

The core app will include styling points, understanding the strain on the apps team, that does not seem to align with our current goals. I would recommend to merge as is, even though it may feel ugly, it is functional, explicit, and easy to read.